### PR TITLE
Express Api - Malin Göthe

### PR DESCRIPTION
**Pull request by M4L5**

### CODE REVIEW

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please add Heroku link for testing 🤓

**Comment by M4L5** [comment]
Hi Karol! Here is the Heroku link: https://goethe-express-deployment.herokuapp.com/avocados

File: README.md [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
-## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
```
Comment (KarolCislowski) [inline README.md:13]: Short documentation would be nice 🤓

File: server.js [hunk 2]
```diff
@@ -1,34 +1,46 @@
...
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('/avocados', (request, response) => {
+  const { region } = request.query
+  if( region ) {
+    const regionList = avocadoSalesData.filter(avocado => avocado.region.includes(region))
+    response.json(regionList)
+  }
+  response.json(avocadoSalesData)
+})
```
Comment (KarolCislowski) [inline server.js:20]: Nice filtering 👍🏻

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**